League Sandbox Auto Setup/leagueSandboxAutoSetupForm.cs

```diff
@@ -251,22 +235,14 @@ private void startDownloadingClient()
         }
         private void startUnzippingClient()
         {
-            if(Directory.Exists(Path.Combine(installDirectoryText.Text, "League of Legends_UNPACKED")))
-            {
-                Directory.Move(Path.Combine(installDirectoryText.Text, "League of Legends_UNPACKED"), Path.Combine(installDirectoryText.Text, "League of Legends"));
-            }
-            if (Directory.Exists(Path.Combine(installDirectoryText.Text, "League of Legends")))
-            {
-                Directory.Move(Path.Combine(installDirectoryText.Text, "League of Legends"), Path.Combine(installDirectoryText.Text, "League_Sandbox_Client"));
-            }
-            if (Directory.Exists(Path.Combine(installDirectoryText.Text, "League_Sandbox_Client")))
+            if (Directory.Exists(Path.Combine(installDirectoryText.Text, "League-of-Legends-4-20")))
```

Comment:
You could set `"League-of-Legends-4-20"` as a class variable/const. Doesn't really matter as the code doesn't really change much.